cairo_programs/compare_lesser_array.cairo

```diff
@@ -0,0 +1,37 @@
+%builtins range_check
+
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.math_cmp import is_le
+
+#Returns TRUE is array_a is lesser than array_b, which means every element in a is lesser than every element in b
```

Comment:
Wouldn't that be lesser than its corresponding element in b?

Reply:
Ouch i've merged and now I saw this message. @fmoletta please send another PR for this